src/segment-loader.js

```diff
@@ -1224,6 +1224,11 @@ export default class SegmentLoader extends videojs.EventTarget {
 
     const Cue = window.WebKitDataCue || window.VTTCue;
     const value = {
+      bandwidth: segmentInfo.playlist.attributes['BANDWIDTH'],
+      resolution: segmentInfo.playlist.attributes['RESOLUTION'],
+      codecs: segmentInfo.playlist.attributes['CODECS'],
+      bitrate: (8 * segmentInfo.byteLength / (end - start)),
```

Comment:
I feel like `bitrate` is a bit ambiguous here. Without seeing the code, I'd probably assume it would be the same thing as `bandwidth`. I think I favour dropping it for clarity's sake since it can be calculated from the other values. Is there a reason you're including it, other than convenience?